obspy/imaging/scripts/mopad.py

```diff
@@ -3554,7 +3554,7 @@ def _check_curve_in_curve(self):
         # correct for ONE special case: double couple with its
         # eigensystem = NED basis system:
         testarray = [1., 0, 0, 0, 1, 0, 0, 0, 1]
-        if np.prod(self.MT._rotation_matrix.A1 == testarray) and \
+        if np.prod(np.ravel(self.MT._rotation_matrix) == testarray) and \
```

Comment:
can this be replaced by `np.allclose(self.Mt._rotation_matrix, test_array) and self.MT._eigenvalues[1] == 0` ? I find the use of `prod` to mean `all` a bit opaque.

Reply:
Unsure about this one. Maybe `np.all` or `all` is equivalent. I was just translating `matrix.A1` to `ravel(array)`.